source/command/msxdos/kmsg.mac

```diff
@@ -97,13 +94,13 @@ CR	EQU	0DH
 
 ;
 	MSG	READING			; Part (c), one leading space.
-	 DEFB	" (�ǂ�",0
+	 DEFB	" (�ǂ�",0

```

Comment:
Are these and the below changes intentional? They look like Japanese text.

Reply:
Hello! That seems to be exactly what happened, it's fixed now. Thank you very much for taking a look and for spotting this!